language-server/src/server.js

```diff
@@ -257,7 +263,11 @@ const getTokenBuilder = (uri) => {
 
 const buildTokens = (builder, document) => {
   const instance = JsoncInstance.fromTextDocument(document);
-  const dialectUri = instance.get("#/$schema").value();
+  if (instance === undefined) {
+    return;
+  }
```

Comment:
Is this necessary? I would expect that this doesn't get called when the document is empty? Did this come up in your exploratory testing?